### DISCUSSION TITLE
IRONING - Paint the areas to be ironed

### DISCUSSION THREAD

**Discussion by fuorissimo** [OrcaSlicer/OrcaSlicer#4867] [Ideas]
It would be very useful to be able to select the areas to iron using the same "painting" procedure present to indicate the areas where to place the supports. 

Otherwise it is not possible to iron specific areas like the one in the photo

![Screenshot (1948)](https://github.com/SoftFever/OrcaSlicer/assets/35644089/4abf4dee-7d4d-4487-a78c-6f2836e03482)

**Comment by convicte** [comment]
I'd like to resurrect this request if possible.
